CONTRIBUTING.md

```diff
@@ -41,65 +43,171 @@ These are the relevant environment variables:
   enabled. If not provided and `WEBHOOK_HOST_LISTEN` is provided,
   it will be defaulted to `WEBHOOK_HOST_LISTEN`.
 
-The Subject Alternative Names of the generated certificate in
-development mode will contain whatever was provided on
-`WEBHOOK_HOST_ADVERTISE` (or whatever it was defaulted to, if it was
-not provided).
+The Subject Alternative Names of the generated certificate in development mode
+will contain whatever was provided on `WEBHOOK_HOST_ADVERTISE` (or whatever it
+was defaulted to, if it was not provided).
+
+To run the controller in the developer workstation it's possible to use
+[telepresence](https://github.com/telepresenceio/telepresence). It allows
+developers to "add" the local workstation in the cluster. Thus, it's possible
+to access workloads in the cluster and receive requests to the controller running
+in the developer machine.  
 
-### Install Custom Resource Definitions
+> Install the telepresence binary from the Github releases page. The official 
+> once from Ambassador Labs may require login. 
 
 Before running the controller, install the custom resource definitions:
 
 ```console
 kubectl apply -f config/crd/bases
 ```
 
-### Create the `kubewarden` namespace
+Create the `kubewarden` namespace
 
 ```console
 kubectl create ns kubewarden
 ```
 
-### Running
+Now, as we are using `telepresence` to intercept the intra cluster
+communication, we need to create the controller deployment and service that
+`telepresence` will intercept. For this, install the Kuberwarden stack
+following the steps described quickstart guide. In other words, install
+Kubewarden controller helm chart. However, a change is required. The controller
+deployment must allow root users. This is disable by default and it is required
+because `telepresence` will add a init container in the deployment which need
+root access.
```

Comment:
```suggestion
deployment must allow root users. This is disabled by default and it is required
because `telepresence` will add a init container in the deployment which needs
root access.
```